com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -603,30 +574,6 @@ var Module = {
         return webgl_support;
     },
 
-    handleVisibilityChange: function () {
```

Comment:
No need for this anymore. It does not capture anything extra that focus+blur events do not already capture.

It also does not have better browser support:

* visibilitychange - https://caniuse.com/?search=visibilitychange
* focus - https://caniuse.com/mdn-api_element_focus_event